### PR TITLE
Delete comment action and links to comment actions in view

### PR DESCRIPTION
**Pull request by nikolatriki**
Notion taks:https://www.notion.so/Comments-delete-action-and-link-to-comment-actions-71fd9bded22945bbb7f98c235cd708c4

### CODE REVIEW

**Review by violeta-p — CHANGES_REQUESTED** [review]
Please add functional specs for this.

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -28,6 +28,13 @@ def update
     end
   end
 
+  def destroy
+    comment = Comment.find(params[:id])
```
Comment (khadzhinov) [inline app/controllers/comments_controller.rb:32]: `comment = Comment.find_by!(giveaway_id: params[:giveaway_id], id: params[:id])`

Comment (nikolatriki) [reply app/controllers/comments_controller.rb:32]: Thanks @khadzhinov, I guess this is the better way beacuse find_by! raises ActiveRecord::RecordNotFound error?

Comment (khadzhinov) [reply app/controllers/comments_controller.rb:32]: Exactly!

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -28,6 +28,13 @@ def update
     end
   end
 
+  def destroy
+    comment = Comment.find_by!(giveaway_id: params[:giveaway_id], id: params[:id])
```
Comment (violeta-p) [inline app/controllers/comments_controller.rb:32]: To DRY the code, we can include destroy action in `before_action :find_comment, only: %i[edit update]`

File: spec/controllers/comments_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                                 comment: { commenter: comment.commenter, body: comment.body } }
+        expect(response).to have_http_status(:found)
+      end
+    end
+  end
+
+  describe 'DELETE destroy' do
+    it 'has a success status code' do
+      delete :destroy, params: { giveaway_id: giveaway.id, id: comment.id }
+      expect(response).to have_http_status(:found)
```
Comment (violeta-p) [inline spec/controllers/comments_controller_spec.rb:48]: I'd suggest using something like this: https://stackoverflow.com/a/19821867/11883546 to test the destroy action.

**Review by violeta-p — APPROVED** [review]

**Pull request merged**